livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -80,6 +80,7 @@ class _TTSOptions:
     api_key: str
     voice: Voice
     model: TTSModels | str
+    language_code: str | None
```

Comment:
I think we should rename language_code to language for consistency

Reply:
renamed

Reply:
fwiw i disagree - language_code is the underlying name in the elevenlabs API